centreon/packages/ui/src/components/Header/PageHeader/PageHeader.styles.ts

```diff
@@ -0,0 +1,105 @@
+import { makeStyles } from 'tss-react/mui';
+
+const useStyles = makeStyles()((theme) => ({
+  header: {
+    alignItems: 'flex-start',
+    borderBottom: `1px solid ${theme.palette.primary.main}`,
+    display: 'flex',
+    flexDirection: 'row',
+    h1: {
+      font: 'normal normal 600 24px/24px Roboto',
+      letterSpacing: '0.18px',
+      margin: theme.spacing(0, 0, 1.5, 0)
+    },
+    justifyContent: 'space-between',
+
+    nav: {
+      display: 'flex',
+      gap: theme.spacing(1),
+      justifyContent: 'flex-end'
+    },
+
+    padding: theme.spacing(0, 0, 1.5, 0)
+  },
+  pageHeader: {
+    alignItems: 'flex-start',
+    borderBottom: `1px solid ${theme.palette.header.page.border}`,
+    display: 'flex',
+    gap: theme.spacing(4),
+    paddingBottom: theme.spacing(2)
+  },
+  pageHeaderActions: {
+    '& > button': {
+      '&:hover': {
+        backgroundColor: theme.palette.header.page.action.background.active,
+        color: theme.palette.header.page.action.color.active
+      },
+      backgroundColor: theme.palette.header.page.action.background.default,
+
+      color: theme.palette.header.page.action.color.default
+    },
+    display: 'flex',
+
+    gap: theme.spacing(2)
+  },
+  pageHeaderMain: {
+    display: 'flex',
+    flexGrow: 1,
+    gap: theme.spacing(1.5)
+  },
+  pageHeaderMenu: {
+    // TEMP
+    '& [data-size="small"]': {
+      height: '20px',
+      marginTop: theme.spacing(0.25),
+      width: '20px'
+    },
+    alignItems: 'top',
+    display: 'flex',
+    opacity: 0.5,
+    padding: theme.spacing(0.25, 0)
+  },
+  pageHeaderTitle: {
+    '& > *': {
+      display: 'grid'
+    },
+    '& > span': {
+      display: 'flex',
+      flexDirection: 'row',
+
+      gap: theme.spacing(2)
+    },
+    display: 'flex',
+
+    flexDirection: 'column',
+
+    gap: theme.spacing(1),
+
+    h1: {
+      color: theme.palette.header.page.title,
+      font: 'normal normal 700 24px/100% Roboto',
+      letterSpacing: '0.15px',
```

Comment:
Why the letterSpacing is different from the other one?
Btw, what do you think about factoring "elements" (like h1, h2 and so on...) styles into a specific CSS provider-like file?

Reply:
we'll need consistency in our 'design system' first (foundations = **typography**, colors... as design tokens)
which should match the `h1/h2...` tag role with the envisioned visual style

till then, the only option is to locally overwrite the global default styles 🤷